custom/icds_reports/migrations/0117_awc_location_local.py

```diff
@@ -5,11 +5,6 @@
 from django.db import migrations, models
 import django.db.models.deletion
 
-from corehq.sql_db.operations import RawSQLMigration
-from custom.icds_reports.utils.migrations import get_view_migrations
-
-migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates', 'database_views'))
-
 
```

Comment:
This is changed to prevent the migration conflict while the build. This wont harm the system either because these views are rebuild in new migrations added